rook/README.md

```diff
@@ -0,0 +1,27 @@
+[![Docker Repository on Quay](https://quay.io/repository/cybozu/rook/status "Docker Repository on Quay")](https://quay.io/repository/cybozu/rook)
+
+Rook container
+==============
+
+This directory provides a Dockerfile to build a rook container that runs [rook][].
+
+Because of the following reasons, the Dockerfile uses [cybozu-go/rook:release][].
+1. The current version of Rook (v1.2.2) does not support partition (`part`) device type, though `master` branch already supported it. So we need to use `master` branch source code.
+2. The current version of Rook (v1.2.2) does not support `topologySpreadConstraints` feature. So we need to patch the original code of Rook.
+
+Until a release from Rook supports the above two features, the following update procedure is needed:
+```
+# Please set $ROOK_VERSION & $MASTER_COMMIT, e.g. ROOK_VERSION="1.1.2"; MASTER_COMMIT="a42d822"
+cd go/src/github.com/rook/rook
+git remote add cybozu-go git@github.com:cybozu-go/rook.git
+git fetch cybozu-go && git checkout release
+git rebase origin/master
+git tag "v$ROOK_VERSION-master-$MASTER_COMMIT"
+git push cybozu-go "v$ROOK_VERSION-master-$MASTER_COMMIT"
+```
+
+After that, please set `TAG` as `$ROOK_VERSION-master` and increment `BRANCH` (i.e. the image tag will be labeled as `v$ROOK_VERSION-master.$BRANCH`). And please update Dockerfile to use the tag `v$ROOK_VERSION-master-$MASTER_COMMIT`.
+
+Note that when a future release from Rook supports the above two features, please modify the update procedure appropriately.
```

Comment:
If the stable version of Rook starts to support the above-mentioned features, please update this procedure too.